src/components/NavBar/account.tsx

```diff
@@ -0,0 +1,71 @@
+import { useRouter } from "next/router";
+
+import ProfilePopover from "@/components/ProfilePopover";
+import { UserRole } from "@/types";
+
+interface Props {
+  role: UserRole;
+  isLoggedIn: boolean;
+  onSignOut: () => void;
+}
+
+const adminRoles = [
+  UserRole.ADMIN,
+  UserRole.ORGANISATION_MANAGER,
+  UserRole.EVENT_MANAGER
+];
+
+export default function Account({
+  role,
+  isLoggedIn,
+  onSignOut
+}: Readonly<Props>) {
+  const router = useRouter();
+
+  return (
+    <div className="flex items-center">
+      {isLoggedIn ? (
+        <>
+          <button
```

Comment:
Consider using the `Link` component provided by nextjs https://nextjs.org/docs/pages/api-reference/components/link which will allow for nice things like preloading the page to be faster